docker/pudl_etl.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
```

Comment:
This whole script kinda feels like it should be a GitHub Action.  Or maybe it's just that GitHub Actions feel like shell scripts.

Reply:
I think you're right. Our full ETL is basically a DAG of python functions and bash commands which will eventually be represented by a Dagster DAG. This bash script works for now but will likely change once we start integrating dagster.